pandas/tests/indexing/test_coercion.py

```diff
@@ -258,6 +258,14 @@ def test_setitem_series_timedelta64(self, val, exp_dtype):
         )
         self._assert_setitem_series_conversion(obj, val, exp, exp_dtype)
 
+    def test_setitem_series_no_coercion_from_values_list(self):
+        # GH35865 - int casted to str when internally calling np.array(ser.values)
+        ser1 = pd.Series(["a", 1])
```

Comment:
can you call one of these 'exp' or 'expected'
then use call the other ser

Reply:
Done!